### ISSUE TITLE
Error to require Prometheus HEAD

### ISSUE THREAD

**Issue by brust** [deadtrickster/prometheus.erl#53]
`mix.exs`
```
defp deps do
  [...
   {:prometheus, github: "deadtrickster/prometheus.erl", ref: "3b0f91f", override: true},
   {:prometheus_ex, "~> 1.1"},
   {:prometheus_ecto, "~> 1.0"},
   {:prometheus_phoenix, "~> 1.0"},
   {:prometheus_plugs, "~> 1.0"},
   {:prometheus_process_collector, "~> 1.0"}
  ]
```

```
$ mix deps.get
$ mix phoenix.server
...
** (Mix) Could not start application my_app: exited in: MyApp.start(:normal, [])
    ** (EXIT) an exception was raised:
        ** (ArgumentError) argument error
            (stdlib) :ets.insert(:prometheus_registry_table, {:default, :prometheus_histogram})
            (prometheus) src/prometheus_registry.erl:79: :prometheus_registry.register_collector/2
            (prometheus) src/prometheus_metric.erl:101: :prometheus_metric.insert_mf/3
            (my_app) lib/my_app/phoenix_instrumenter.ex:2: MyApp.PhoenixInstrumenter.setup/0
            (my_app) lib/my_app.ex:21: MyApp.start/2
            (kernel) application_master.erl:273: :application_master.start_it_old/4
```

```
$ iex
Erlang/OTP 19 [erts-8.1] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Interactive Elixir (1.3.4) - press Ctrl+C to exit (type h() ENTER for help)
iex(1)> :os.type
{:unix, :darwin}
```

**Comment by brust** [comment]
#52

**Comment by deadtrickster** [comment]
Hi, Could you please test with 3.2.2 now?

**Comment by deadtrickster** [comment]
btw if you use `:prometheus_ex` there is no need to include `:prometheus`. IIRC it's all started in #52 - when you were not able to use `:prometheus` directly

**Comment by brust** [comment]
Fixed on `3.2.2`
